src/Perf.h

```diff
@@ -0,0 +1,78 @@
+#ifndef PERF_H
+#define PERF_H
+
+#include "AtomicMCDAObject.h"
+#include <iostream>
+#include <string>
+
+class Perf : public AtomicMCDAObject {
+public:
+  /**
+   * Perf standard constructor
+   *
+   * @param name name of the performance (ex: name of profile or alternative)
+   * @param criterion criterion id on which the performance in based on
+   * @param value value of the performance
+   */
+  Perf(std::string name, std::string criterion, float value);
```

Comment:
What is better float or doubles for precision and memory allocation ?

Reply:
Precision --> double
memory & vectorisation --> float
I asked Vincent  in June : float are more than enough for the precision